### ISSUE TITLE
Hash check incomplete?

### ISSUE THREAD

**Issue by tuxuser** [MakeMHz/xenium-fw-update#3]
In the following section:

https://github.com/tuxuser/xenium-fw-update/blob/044cc0e16661a8a9909ecd8cf849ac104fdfd519/src/main.cpp#L229-L234

```c
            // Check to see this matches any known values
            for (uint32_t i = 0; i < (sizeof(XeniumVersions) / sizeof(Version)); i++) {
                if (XeniumVersions[i].crc32_bootloader == bank_bootloader_crc32 &&
                    XeniumVersions[i].crc32_bootloader == bank_bootloader_crc32) {
                    // Store reference
                    matched_version = (Version *)&XeniumVersions[i];
```

it looks like the section should be modified like this, correct?

```diff
-                    XeniumVersions[i].crc32_bootloader == bank_bootloader_crc32) {
+                    XeniumVersions[i].crc32_xos == bank_xeniumos_crc32) {
```

**Comment by LoveMHz** [comment]
Copy and paste mistake. Good catch!

**Comment by tuxuser** [comment]
Fixed with 208115b04a252bf4a8ed5f7d125beb049cf038c5

**Comment by LoveMHz** [comment]
Oh, whoops! Thanks, @tuxuser! I completely forgot about this issue. PRs are more than welcome in the future.
